pwndbg/gdblib/config.py

```diff
@@ -95,6 +95,11 @@ def __get_set_string_gdb_gte_9(self) -> str:
             self.param.value = self.value
 
         for trigger in config.triggers[self.param.name]:
+            # validate param value before trigger
+            if not config.validate_choices(self.param.name, self.param.value):
+                self.param.revert_default()
+                return f"Invalid value '{self.param.value}' for parameter '{self.param.name}'. Valid choices are: {', '.join(self.param.choices)}"
```

Comment:
Does this if need to be in the trigger loop? The condition do not use `trigger` at all and here we will be re-validating things on each trigger. Is that needed?